lib/infra/infra-stack.ts

```diff
@@ -585,7 +586,25 @@ export class InfraStack extends Stack {
         }));
     }
 
-    // // Startinng OpenSearch based on whether the distribution type is min or bundle
+    if (props.customConfigFiles !== 'undefined') {
+      try {
+        // @ts-ignore
+        const jsonObj = JSON.parse(props.customConfigFiles);
+        Object.keys(jsonObj).forEach((localFileName) => {
+          const getConfig = load(readFileSync(localFileName, 'utf-8'));
+          const remoteConfigLocation = jsonObj[localFileName];
```

Comment:
I am assuming this provides the value, i.e, the actual path of the file in opensearch or opensearch-dashboards directory on the ec2 host?

Reply:
Yes! root directory remains the same which is `/home/ec2-user`, the value of json needs to be `opensearch/config/*` etc.

Reply:
Also, for local file name, the user has to provide the absolute path or does the code reads using relative-path?

Reply:
Both works as per my testing. Relative as well as absolute path.